src/pages/index.js

```diff
@@ -115,6 +106,12 @@ export default ({ location }) => {
                   </ContactInfo>
                 </div>
               </div>
+              <div className="py-2">
+                <h5>Patrica Pistorio</h5>

```

Comment:
Spelling

Reply:
That's the spelling the Initial Description had. Unless it was a error?
![Capture](https://user-images.githubusercontent.com/17188237/192365386-0c92de96-8d56-46a8-a029-02a7e4c1f871.PNG)
@casjr13

Reply:
```
Yours says:     `Patrica` 
Supposed to be: `Patricia`
```